stories/topics/tech-note-aws-backup-restore.adoc

```diff
@@ -22,6 +22,16 @@
 * `<stack_name>-pulp_cert`
 * `<stack_name>-pulp_key`
 
+. The Ansible on Clouds Ops Container restore playbook fails when attempting to restore a secret starting with `-`. If the restore operation is failing, on your deployment please ensure the secret `<stack_name>-database_fields_symmetric_key` does not start with a `-`. Follow the steps below to check for this case and update the secret appropriately.
+.. Find the backup you are passing to the restore playbook on S3.
+... The `aws_backup_name` parameter you passed to the restore playbook is the backup we are trying to locate and it should be a folder inside of an S3 bucket
+... The `aws_s3_bucket` parameter you passed to the restore playbook should be the bucket which contains the backup we are trying to locate
+.. After locating the backup, navigate inside of the backup folder and download the `secrets.json` file.
```

Comment:
One period for each of these.

Reply:
<img width="711" alt="Screenshot 2023-07-27 at 11 59 20 AM" src="https://github.com/ansible/aap-docs/assets/112966709/daf1e4fb-6cca-45ef-92ed-a28143723184">